uefi/biosregion.go

```diff
@@ -54,3 +55,25 @@ func NewBIOSRegion(buf []byte, r *Region) (*BIOSRegion, error) {
 	}
 	return &br, nil
 }
+
+// Extract extracts the Bios Region to the directory passed in.
+func (br *BIOSRegion) Extract(dirPath string) error {
+	// Create the directory if it doesn't exist
+	err := os.MkdirAll(dirPath, 0755)
+	if err != nil {
+		return err
+	}
+
+	// Dump the binary.
+	br.ExtractPath = dirPath + "/biosregion.bin"
```

Comment:
I think it should be a directory, because extracting this bios region may result in multiple files. For example if we want to extract the firmware volumes later, I don't want to have to pass in multiple filenames or have to remove the "biosregion.bin" that was passed in to call the sub directories. 

Good point on the path.join though. I'll change that.